lib/entry/LiveEntry.js

```diff
@@ -130,6 +131,23 @@ function saveFlavorDownloadersConfiguration() {
     });
 }
 
+function getEntryServerNodeId() {
+    let entryServerNodeFilter = new kalturaVO.KalturaEntryServerNodeFilter();
+    entryServerNodeFilter.serverTypeEqual = this.entryServerType;
+    var getLiveEntryServerNodeIdPromise =
```

Comment:
i think that the call to the API should be BackendClient.getEntryServerNodeByType(this.entryId,this.entryServerType)
(rename getSingleEntryServerNode to getEntryServerNodeByType and  move the KalturaEntryServerNodeFilter into inside the getEntryServerNodeByType function)

this is because the filter is internal implmentation of the backendclient